strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/storage/repository/Repository.java

```diff
@@ -67,6 +67,9 @@
     @XmlAttribute(name = "checksum-headers-enabled")
     private boolean checksumHeadersEnabled;
 
+    @XmlElement(name = "semantic-versioning")
```

Comment:
I don't think this is the way this should be controlled... @fuss86 recently implemented a way to define the validators used by a repository. This can be controlled via the `strongbox.xml`. You can have a look at #332 .

Reply:
yes, I saw this but I thought that this is quite an important option and it deserves to be dedicated explicitly

Reply:
this option will also enable semver validator